src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -487,13 +491,13 @@ private void DoStart()
             if (logger.IsVerbose) { logger.Verbose("Storage provider manager created successfully."); }
 
             // Load and init stream providers before silo becomes active
-            var siloStreamProviderManager = (StreamProviderManager) grainRuntime.StreamProviderManager;
+            var streamProviderManager = (StreamProviderManager)grainRuntime.StreamProviderManager;
```

Comment:
why do you need to rename that? Lets keep unrelated churn low. If you want to refactor, unrelated to the change, please submit it is a separate PR.